ingest/indexer/service/client/pubsub_client.go

```diff
@@ -82,6 +82,10 @@ func (p *PubSubClient) PublishPool(ctx context.Context, pool indexerdomain.Pool)
 	return p.publish(ctx, pool)
 }
 
+func (p *PubSubClient) PublishTokenSupply(ctx context.Context, tokenSupply indexerdomain.TokenSupply) error {
+	return p.publish(ctx, tokenSupply)
+}
```

Comment:
Note: I created a separate API for pushing token supply

Reply:
I was actually thinking about token supply will be one of the attributes in Asset struct and be pushed to to asset_topic ?

Reply:
Got it. I suspected that was the plan - let's discuss during our upcoming plan